components/library/qt5/Makefile

```diff
@@ -21,17 +22,17 @@ include ../../../make-rules/shared-macros.mk
 
 COMPILER=gcc
 COMPONENT_NAME=		qt
-COMPONENT_VERSION=	5.12.10
-COMPONENT_VERSION_MJR=	5.12
-COMPONENT_REVISION=     3
+COMPONENT_VERSION=	5.15.3
+COMPONENT_VERSION_MJR=	5.15
+COMPONENT_REVISION=     1
 COMPONENT_FMRI=		library/qt5
```

Comment:
You don't need to set COMPONENT_REVISION if no release for COMPONENT_VERSION has happened before.